react-landing/src/App.jsx

```diff
@@ -1,12 +1,23 @@
 import './App.css';
+import ProductItem from './components/ProductItem';
 import ProductList from './components/ProductList';
-/* import data from './data/data.json'; */
+import data from './data/data.json';
 
 export default function App() {
+
+  const productGrid = data.map((product, index)=> {
+    return (
+      <div className="col-span-2" key={index} >
+        <ProductItem product={product}/>
+      </div>
+    )
+    });
```

Comment:
- Busca separar esto en otro componente para que se siga más el concepto de "separation of concerns." https://www.freecodecamp.org/news/separation-of-concerns-react-container-and-presentational-components/
- Si no separas las responsabilidades, busca mantener más consistencia en espaciado (líneas 9 a 13 deberían estar un tab más a la derecha). Para esto puedes usar herramientas como Prettier que se ejecutan automáticamente